website/docs/r/firewall.html.md

```diff
@@ -46,7 +46,7 @@ The following arguments are supported:
 
 * `rules_v4_in` - (Optional*) Firewall template rules for inbound traffic - covers ipv4 addresses.
 
-    * `order` - (Required) The order at which the firewall will compare packets against its rules, a packet will be compared against the first rule, it will either allow it to pass or block it and it won t be matched against any other rules. However, if it does no match the rule, then it will proceed onto rule 2. Packets that do not match any rules are blocked by default.
+    * `order` - (Required) The order at which the firewall will compare packets against its rules, a packet will be compared against the first rule, it will either allow it to pass or block it and it won t be matched against any other rules. However, if it does no match the rule, then it will proceed onto rule 2.
```

Comment:
Let's make a sentence here: `The order at which the firewall will compare packets against its rules. A packet will be…`